pkg/otlp/metrics/metrics_translator.go

```diff
@@ -716,6 +705,16 @@ func (t *Translator) MapMetrics(ctx context.Context, md pmetric.Metrics, consume
 			newMetrics := pmetric.NewMetricSlice()
 			for k := 0; k < metricsArray.Len(); k++ {
 				md := metricsArray.At(k)
+				if md.Name() == keyStatsPayload {
+					// these metrics are an APM Stats payload; consume it as such
+					for l := 0; l < md.Sum().DataPoints().Len(); l++ {
+						if payload, ok := md.Sum().DataPoints().At(l).Attributes().Get(keyStatsPayload); ok && t.cfg.statsOut != nil {
+							t.cfg.statsOut <- payload.Bytes().AsRaw()
```

Comment:
This will panic if the attribute is not `pcommon.Bytes`. Same comment as above, let's try to avoid panics